_locales/de/messages.json

```diff
@@ -789,11 +789,14 @@
     "messageformat": "Es gab einen Fehler bei der Sprachaufzeichnung."
   },
   "icu:attachmentSaved": {
-    "messageformat": "Anhang gespeichert."
+    "messageformat": "{count, plural, one {Anhang} other {# Anhänge}} gespeichert."
   },
   "icu:attachmentSavedShow": {
     "messageformat": "Im Ordner anzeigen"
   },
+  "icu:attachmentStillDownloading": {
+    "messageformat": "{count, plural, one {Anhang kann} other {Anhänge können}} nicht gespeichert werden, da {count, plural, one {er} other {#}} noch nicht fertig heruntergeladen {count, plural, one {wurde} other {wurden}}."
+  },
```

Comment:
We don't modify the non-english files - those are managed by our translators. This will be overwritten the next time we pull strings.

Reply:
Okay, I wasn't sure how to interpret [this](https://github.com/signalapp/Signal-Desktop/blob/main/CONTRIBUTING.md#pull-requests) "You only need to modify the default locale [_locales/en/messages.json](https://github.com/signalapp/Signal-Desktop/blob/main/_locales/en/messages.json)." statement.
I thought this means that I have to modify the English file, but can edit other languages as well.
In my case this was necessary, because I had the Signal instance running in German and it crashed when the translated string wasn't present.

But if I understand you correctly, this means that I can leave the string as is?